cumulus_library/builders/statistics_templates/counts_templates.py

```diff
@@ -46,13 +46,16 @@ class CountAnnotation:
     :keyword join_field: the column in the table to join on
     :keyword list: a list of tuples like ('column_name', 'alias' or None) to
         define columns to add to your dataset.
+    :keyword alt_target: if present, a column to use from the annotation table
+        instead of the one defined in `field` as the primary countable target
```

Comment:
Is this description accurate? If this is defined, *all* columns from the annotation become the primary countable target. The result is merely _sorted_ by `alt_target`.